### ISSUE TITLE
[201911] fast-reboot fail on fast-reboot-dump.py script

### ISSUE THREAD

**Issue by prsunny** [Azure/sonic-buildimage#7736]
#### Description
fast-reboot-dump.py script failed with exception

```
May 20 17:23:39.124041 str-msn2700-22 INFO fast-reboot-dump: Neighbor entry: [Vlan: Vlan1000, Mac: 72:06:00:01:04:30, Ip: 192.168.1.176]
May 20 17:23:39.285529 str-msn2700-22 ERR fast-reboot-dump: Got an exception '192.168.1.210' does not appear to be an IPv4 or IPv6 interface: Traceback: Traceback (most recent call last):#012 
File "/usr/bin/fast-reboot-dump.py",line 340,in <module>#012    res = main()#012  
File "/usr/bin/fast-reboot-dump.py",line 333,in main#012    send_garp_nd(neighbor_entries,map_mac_ip_per_vlan)#012  
File "/usr/bin/fast-reboot-dump.py", line 277, in send_garp_nd#012    if ipaddress.ip_interface(dst_ip).ip.version == 4:#012  
File "/usr/local/lib/python2.7/dist-packages/ipaddress.py", line 239, in ip_interface#012    address)#012ValueError: '192.168.1.210' does not appear to be an IPv4 or IPv6 interface
May 20 17:23:39.291055 str-msn2700-22 NOTICE admin: fast-reboot failure (0) cleanup ...
```

This was introduced by PR https://github.com/Azure/sonic-utilities/pull/1047 while cherry-picked to 201911
Fix provided in - https://github.com/Azure/sonic-utilities/pull/1627

#### Steps to reproduce the issue:
1. Run fast-reboot command
2.
3.

#### Describe the results you received:
fast-reboot failed with exception

#### Describe the results you expected:
Pass

#### Output of `show version`:

```
SONiC.20191130.70 and above
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by prsunny** [comment]
@rlhui , @sumukhatv for viz

**Comment by qiluo-msft** [comment]
Is the issue closed by https://github.com/Azure/sonic-utilities/pull/1627?

**Comment by prsunny** [comment]
> Is the issue closed by [Azure/sonic-utilities#1627](https://github.com/Azure/sonic-utilities/pull/1627)?

Yes
